daprdocs/content/en/operations/observability/metrics/metrics-overview.md

```diff
@@ -80,6 +80,14 @@ The default value of `spec.metrics.http.increasedCardinality` is `true` in Dapr
 
 Setting `spec.metrics.http.increasedCardinality` to `false` is **recommended** to all Dapr users, to reduce resource consumption. The pre-1.13 behavior, which is used when the option is `true`, is considered legacy and is only maintained for users who have special requirements around backwards-compatibility.
```

Comment:
Do we still want to say these words, that this is consider legacy?

Reply:
should we invert the wording? Something like:

Starting with version 1.14, the default setting for spec.metrics.http.increasedCardinality is false to reduce resource consumption. The previous behavior, where this option is set to true, is considered legacy and is maintained only for users with specific backward-compatibility requirements.